docs/flow/modin/experimental/core/execution/ray/implementations/pandas_on_ray/index.rst

```diff
@@ -0,0 +1,24 @@
+:orphan:
+
+Experimental PandasOnRay Execution
+==================================
+
+Experimental PandasOnRay execution is designed to leave the underlying mechanisms of PandasOnRay
+execution: "Data Transformation", "Data Ingress", "Data Egress" architecturally
+:doc:`unchanged </flow/modin/core/execution/ray/implementations/pandas_on_ray/index>`.
+
+Main differencies between these two executions
+----------------------------------------------
+- new Factory ``PandasOnRayFactory`` -> ``ExperimentalPandasOnRayFactory``
+- new IO class ``PandasOnRayIO`` -> ``ExperimentalPandasOnRayIO``
+
+Objects implementations for Experimental PandasOnRay IO
+-------------------------------------------------------
+- :py:class:`~modin.experimental.core.execution.ray.implementations.pandas_on_ray.io.io.ExperimentalPandasOnRayIO`
+- :py:class:`~modin.core.execution.dispatching.factories.factories.ExperimentalPandasOnRayFactory`
+- :py:class:`~modin.core.io.text.csv_glob_dispatcher.CSVGlobDispatcher`
+- :py:class:`~modin.core.storage_formats.pandas.parsers.PandasCSVGlobParser`
```

Comment:
It seems that these links are not working because `.. autoclass::` directives are missed for the classes.

I see no reason to add pictures here as they are architecturally the same as for `PandasOnRay` execution. @YarShev are you ok with that?

Reply:
> It seems that these links are not working because .. autoclass:: directives are missed for the classes.

We have other classes too which we do not generate documentation for. I think we should do that at some point.

> I see no reason to add pictures here as they are architecturally the same as for PandasOnRay execution. @YarShev are you ok with that?

Yes, I am ok with that, a link to PandasOnRay execution should be enough.